source/includes/_simulator-guide.html.md

```diff
@@ -0,0 +1,130 @@
+
+
+## Proxy Support
+> Example Python Proxy
+
+```python
+my_config.proxy == 'myproxy:5000'
+```
+
+> Example CLI Proxy Usage
+
+```shell
+$ python <simulator_file>.py --proxy=localhost:3128
+```
+
+To connect a simulator from behind a corporate proxy, command line tools like the Bonsai SDK need to know where to send traffic. For the Bonsai SDK, this is done by providing the proxy server address and port to the simulator config when running a simulator. There are multiple ways that you can do this.
+## Prediction
+
+```python
+# Default version is latest
+python <simulator_file>.py --predict
+
+#OR
+Python <simulator_file>.py --predict=1
+```
+
+After your BRAIN is finished training it can be used to "predict" or perform in the simulation. How well it does depends on how long you let it train! Using your BRAIN involves running your simulator file as you did when training, but now in prediction mode with `--predict`. The default is `--predict=latest` which will use the version of the latest training session that you just ran. You can use a different version of your BRAIN if you have trained it multiple times by replacing `latest` with the version number.
+
+# Cloud Hosted Simulators
+There are a couple simulators that the Bonsai Platform can run cloud-hosted (sometimes referred to as platform managed) in a Docker container. This means you can start and stop training all from the web interface, without needing to download and install the Bonsai CLI or SDK. You will still need to run the simulator locally using the CLI to obtain predictions, however.
+
+Cloud-hosted simulators are useful for prolonged training to avoid your computer becoming idle and stopping training. At this time, you can only run one simulator cloud-hosted, and cannot take advantage of training speed increases from [running simulators in parallel][5].
+
+Current list of supported simulators for Docker cloud-hosted training:
+
+* OpenAI Gym environment: [`openai.gym`][1]
+* Bonsai Python `bonsai-ai` library: [`bonsai.ai`][10]
+* EnergyPlus Simulator: [`bonsai.energyplus`][3]
+* Legacy `bonsai-python` library: [`bonsai.python`][2]
+
+In order to set your BRAIN to use one of these cloud-hosted simulators, you’ll need to modify your BRAIN’s [project (`.bproj`) file][8] if you aren’t already using one of the Bonsai demo projects.
+
+# Running Simulations in Parallel
+
+> ![Parallel Simulators](../images/multiple-sims.png)
+
+Training can take a long time. Depending on the complexity of the problem and they way your lesson plans are written, training can take several hours to several days. Fortunately, training can happen in parallel. Each new simulator you connect to a running training session will increase the Episodes/Sec training rate which will help the BRAIN find better solutions faster. How much you can increase training depends on the speed of your simulation: if you simulation takes 1s per iteration, you will get benefits running dozens
```

Comment:
last sentence not finished. 

Mention that eventually, system can't handle any more messages per second.